util/crc32c_test.cc

```diff
@@ -137,6 +137,22 @@ TEST(CRC, Mask) {
   ASSERT_EQ(crc, Unmask(Unmask(Mask(Mask(crc)))));
 }
 
+TEST(CRC, Crc32cCombineTest1) {
```

Comment:
Please add more tests. We should probably cover all sizes of crc2 from 0 to 1000. Perhaps also one big size like 2^24 - 1 (for good coverage of the modular exponentiation).

Reply:
Sure, will add the testing